src/include/duckdb/main/extension_entries.hpp

```diff
@@ -20,208 +20,209 @@ struct ExtensionEntry {
+    {"st_intersects_extent", "spatial", "scalar"},
+    {"st_isclosed", "spatial", "scalar"},
+    {"st_isempty", "spatial", "scalar"},
+    {"st_isring", "spatial", "scalar"},
+    {"st_issimple", "spatial", "scalar"},
+    {"st_isvalid", "spatial", "scalar"},
+    {"st_length", "spatial", "scalar"},
+    {"st_length_spheroid", "spatial", "scalar"},
+    {"st_linemerge", "spatial", "scalar"},
+    {"st_linestring2dfromwkb", "spatial", "scalar"},
+    {"st_list_proj_crs", "spatial", "table"},
+    {"st_makeenvelope", "spatial", "scalar"},
+    {"st_makeline", "spatial", "scalar"},
+    {"st_makepolygon", "spatial", "scalar"},
+    {"st_makevalid", "spatial", "scalar"},
+    {"st_ngeometries", "spatial", "scalar"},
+    {"st_ninteriorrings", "spatial", "scalar"},
+    {"st_normalize", "spatial", "scalar"},
+    {"st_npoints", "spatial", "scalar"},
+    {"st_numgeometries", "spatial", "scalar"},
+    {"st_numinteriorrings", "spatial", "scalar"},
+    {"st_numpoints", "spatial", "scalar"},
+    {"st_overlaps", "spatial", "scalar"},
+    {"st_perimeter", "spatial", "scalar"},
+    {"st_perimeter_spheroid", "spatial", "scalar"},
+    {"st_point", "spatial", "scalar"},
+    {"st_point2d", "spatial", "scalar"},
+    {"st_point2dfromwkb", "spatial", "scalar"},
+    {"st_point3d", "spatial", "scalar"},
+    {"st_point4d", "spatial", "scalar"},
+    {"st_pointn", "spatial", "scalar"},
+    {"st_pointonsurface", "spatial", "scalar"},
+    {"st_polygon2dfromwkb", "spatial", "scalar"},
+    {"st_quadkey", "spatial", "scalar"},
+    {"st_read", "spatial", "table"},
+    {"st_read_meta", "spatial", "table"},
+    {"st_readosm", "spatial", "table"},
+    {"st_readshp", "spatial", "table"},
+    {"st_reduceprecision", "spatial", "scalar"},
+    {"st_removerepeatedpoints", "spatial", "scalar"},
+    {"st_reverse", "spatial", "scalar"},
+    {"st_simplify", "spatial", "scalar"},
+    {"st_simplifypreservetopology", "spatial", "scalar"},
+    {"st_startpoint", "spatial", "scalar"},
+    {"st_touches", "spatial", "scalar"},
+    {"st_transform", "spatial", "scalar"},
+    {"st_union", "spatial", "scalar"},
+    {"st_union_agg", "spatial", "aggregate"},
+    {"st_within", "spatial", "scalar"},
+    {"st_x", "spatial", "scalar"},
+    {"st_xmax", "spatial", "scalar"},
+    {"st_xmin", "spatial", "scalar"},
+    {"st_y", "spatial", "scalar"},
+    {"st_ymax", "spatial", "scalar"},
+    {"st_ymin", "spatial", "scalar"},
+    {"stem", "fts", "scalar"},
+    {"text", "excel", "scalar"},
+    {"to_arrow_ipc", "arrow", "table"},
+    {"to_json", "json", "scalar"},
+    {"tpcds", "tpcds", "pragma"},
+    {"tpcds_answers", "tpcds", "table"},
+    {"tpcds_queries", "tpcds", "table"},
+    {"tpch", "tpch", "pragma"},
+    {"tpch_answers", "tpch", "table"},
+    {"tpch_queries", "tpch", "table"},
+}; // END_OF_EXTENSION_FUNCTIONS
 
 static constexpr ExtensionEntry EXTENSION_SETTINGS[] = {
     {"azure_storage_connection_string", "azure"},
-    {"binary_as_string", "parquet"},
```

Comment:
Not sure what happened here?
I built with `DISABLE_BUILTIN_EXTENSIONS=1 BUILD_ALL_EXT=1 make release` as I have also added to the script as an error message when the prerequisites arent met

Reply:
I have manually added the missing setting from the parquet extension for now, we should probably update the notes in the generation script about this though